.pre-commit-config.yaml

```diff
@@ -1,16 +1,17 @@
+---
```

Comment:
can you remove this file from the changeset? I've pulled the latest esphome into dev and replaced this MR on top of that.

Reply:
Resolved with 2d665e8e57f9c39d43d03a8385f3943d80de7e7b